### PR TITLE
DSERV-257 load EBI protein complex

### PR DESCRIPTION
**Pull request by shengchd**
- Adapter for loading EBI protein complex from tsv file
- Script for generating dict containing binding regions from complex API

### CODE REVIEW

File: data/data_loading_support_files/EBI_complex/get_EBI_complex_linkedFeatures.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import requests
+import csv
+import pickle
+
+# pull binding regions info from api for each complex (which is not available from the tab file downloaded from the portal).
+
+
+def get_binding_regions(complex_id):
+    url = 'https://www.ebi.ac.uk/intact/complex-ws/complex/' + complex_id
+    complex_json = response = requests.get(url).json()
```
Comment (pedrohr) [inline data/data_loading_support_files/EBI_complex/get_EBI_complex_linkedFeatures.py:10]: the `complex_json =` can be removed

**Review by pedrohr — APPROVED** [review]

**Comment by shengchd** [comment]
Removed complex-complex loading part for DSERV-287.

File: data/dev_setup.py [hunk 2]
```diff
@@ -74,6 +75,10 @@
     'oncotree_terms': Oncotree(type='node'),
     'oncotree_relationships': Oncotree(type='edge'),
     'gene_term': DepMap('./samples/DepMap/CRISPRGeneDependency_transposed_example.csv', type='edge', label='gene_term'),
+    'complex': EBIComplex('./samples/EBI_complex.tsv', label='complex'),
+    'complex_protein': EBIComplex('./samples/EBI_complex.tsv', label='complex_protein'),
+    'complex_term': EBIComplex('./samples/EBI_complex.tsv', label='complex_term'),
+    'complex_complex': EBIComplex('./samples/EBI_complex.tsv', label='complex_complex'),
```
Comment (pedrohr) [inline data/dev_setup.py:81]: this line can be removed too

**Review by pedrohr — CHANGES_REQUESTED** [review]
just one line to be removed in the dev_setup script

**Review by pedrohr — APPROVED** [review]

**Pull request merged**